core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -225,9 +225,22 @@ private[spark] class CoarseMesosSchedulerBackend(
             .addResources(createResource("cpus", cpusToUse))
             .addResources(createResource("mem",
               MemoryUtils.calculateTotalMemory(sc)))
-            .build()
+
+          sc.conf.getOption("spark.executor.docker.image").map { image:String =>
+            val container = task.getContainerBuilder()
+            val volumes = sc.conf
+              .getOption("spark.executor.docker.volumes")
+              .map(MesosSchedulerBackendUtil.parseVolumesSpec _)
```

Comment:
I think you can do without the `_` here (and other places). Correct me if I'm wrong.